### PR TITLE
Docs for contracts:items:unit:value

### PR DESCRIPTION
**Pull request by ivanromakh**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/openprocurement.tender.limited/65)

### CODE REVIEW

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -194,6 +194,14 @@ Setting contract validity period is optional, but if it is needed, you can set a
 .. include:: tutorial/tender-contract-period.http
    :code:
 
+Setting contract value for unit
+-------------------------------
+
+Setting contract value for unit is optional.
```
Comment (vmaksymiv) [inline docs/source/tutorial.rst]: Additionally you can set unit value for contract item.

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -194,6 +194,14 @@ Setting contract validity period is optional, but if it is needed, you can set a
 .. include:: tutorial/tender-contract-period.http
    :code:
 
+Setting contract value for unit
```
Comment (vmaksymiv) [inline docs/source/tutorial.rst]: Setting unit value for item

**Comment by ivanromakh** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[docs/source/tutorial.rst, line 197 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.tender.limited/65#-KYDd166NTSsUqHi46Se-r1-197:-KYOb3Z61XUx3nPXv9ri:bp3ramz) ([raw file](https://github.com/openprocurement/openprocurement.tender.limited/blob/920e0219a0cd7e7eafca0e325c477acf481970cd/docs/source/tutorial.rst#L197)):*
<details><summary><i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary><blockquote>

Setting unit value for item
</blockquote></details>

зробив

---

*[docs/source/tutorial.rst, line 200 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.tender.limited/65#-KYDd166NTSsUqHi46Se-r1-200:-KYOb51e8-Zbtbv5n1qv:bp3ramz) ([raw file](https://github.com/openprocurement/openprocurement.tender.limited/blob/920e0219a0cd7e7eafca0e325c477acf481970cd/docs/source/tutorial.rst#L200)):*
<details><summary><i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary><blockquote>

Additionally you can set unit value for contract item. 
</blockquote></details>

зробив

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.tender.limited/65)*

**Comment by vmaksymiv** [comment]
Зліпи всі коміти в один.

**Pull request closed without merging**